instance/models/mixins/openedx_database.py

```diff
@@ -95,6 +95,10 @@ def mysql_databases(self):
                     }
                 ],
             },
+            {
+                "name": self._get_mysql_database_name("programs"),
+                "user": self._get_mysql_user_name("program"),
+            },
```

Comment:
Also looks like we can remove `mongo_provisioned`, because if [`mongo.add_user`](https://github.com/open-craft/opencraft/blob/f94660e9e267f85af61ceb50c6600003998b6276/instance/models/mixins/database.py#L223) is called on an existing user, all it does is potentially [update the password](http://api.mongodb.com/python/current/api/pymongo/database.html#pymongo.database.Database.add_user).

And [`swift_provisioned`](https://github.com/open-craft/opencraft/blob/077e1a71dcfc574581b4c11581794d9b8436d45b/instance/models/mixins/storage.py#L59) too!  If the container already exists, it might only [update its metadata](http://docs.openstack.org/developer/python-swiftclient/swiftclient.html#swiftclient.client.post_container).

Does that sound ok to you @mtyaka ?